src/mame/drivers/firebeat.cpp

```diff
@@ -1195,6 +1424,24 @@ TIMER_DEVICE_CALLBACK_MEMBER(firebeat_spu_state::spu_timer_callback)
 /*****************************************************************************
 * beatmania III
 ******************************************************************************/
+static void pc_hd_floppies(device_slot_interface &device)
+{
+	device.option_add("35hd", FLOPPY_35_HD);
+}
+
+FLOPPY_FORMATS_MEMBER(firebeat_bm3_state::floppy_formats)
+	FLOPPY_PC_FORMAT
+FLOPPY_FORMATS_END
```

Comment:
Could you merge latest master and make sure it still builds?  I think these things have turned into functions now (see 3c9e3749541b1da8a0c5409c582070aae66dd3d6).

Reply:
Oh, maybe that's the issue then. Sure, I'll try that out.

Reply:
That was exactly it. I finally started getting the same build errors locally and was able to fix the build errors. Thanks!